lib/tokenize.js

```diff
@@ -5,10 +5,24 @@
  *
  * @return  {array}
  */
-module.exports = function (input) {
-    return input
-        .toLowerCase()
-        .replace(/[^a-z0-9á-úñäâàéèëêïîöôùüûœç\-\' ]+/g, '')
-        .replace('/ {2,}/',' ')
-        .split(' ');
+var spliddit = require('spliddit');
+
+module.exports = function(input) {
+    if (input === undefined) return [];
+    if (input.trim() === '') return [''];
```

Comment:
Is your thinking that this is a performance optimization? If so, I think I'd rather track it in a separate PR.